clearwater-infrastructure/usr/share/clearwater/infrastructure/scripts/1hosts

```diff
@@ -52,6 +53,9 @@ then
   echo $local_ip $hostname '# maps to local IPv6 address' >> /tmp/hosts.$$
 fi
 
+# Append the IPv6 localhost
```

Comment:
Might be worth referencing the RFC information from https://github.com/Metaswitch/sprout/issues/1510#issuecomment-240825915 in a comment here, so we know we've got a sound reason for doing this if we look at this in future. Particularly the "Users may assume that IPv4 and IPv6 address queries for localhost names will always resolve to the respective IP loopback address" stuff.